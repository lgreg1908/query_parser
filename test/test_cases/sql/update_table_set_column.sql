UPDATE table SET column1 = 10