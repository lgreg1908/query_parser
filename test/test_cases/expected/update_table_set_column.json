{
    "type": "ROOT",
    "children": [
        {"type": "DML", "value": "UPDATE", "is_group": false},
        {"type": "Identifier", "value": "table", "is_group": false},
        {"type": "Keyword", "value": "SET", "is_group": false},
        {"type": "Identifier", "value": "column1", "is_group": false},
        {"type": "Comparison", "value": "= 10", "is_group": false}
    ]
}